tests/integration/ha_tests/application-charm/src/continuous_writes.py

```diff
@@ -17,22 +37,14 @@ def continuous_writes(connection_string: str, starting_number: int):
     """
     write_value = starting_number
 
-    try:
-        # Create the table to write records on and also a unique index to prevent duplicate writes.
-        with psycopg2.connect(connection_string) as connection, connection.cursor() as cursor:
-            connection.autocommit = True
-            cursor.execute("CREATE TABLE IF NOT EXISTS continuous_writes(number INTEGER);")
-            cursor.execute(
-                "CREATE UNIQUE INDEX IF NOT EXISTS number ON continuous_writes(number);"
-            )
-    finally:
-        connection.close()
+    read_config_file()
 
     # Continuously write the record to the database (incrementing it at each iteration).
-    while True:
+    while run:
         try:
             with psycopg2.connect(connection_string) as connection, connection.cursor() as cursor:
                 connection.autocommit = True
+                cursor.execute("SET synchronous_commit = 'remote_apply';")
```

Comment:
I think this line is not needed in the test 😞 it should be controlled on postgresql side 😞